flink-clients/src/test/java/org/apache/flink/client/ExecutionEnvironmentTest.java

```diff
@@ -91,9 +106,42 @@ public void testUserDefinedJobNameWithConfigure() {
         testJobName(jobName, env);
     }
 
+    @Test
+    public void testConfigureFileSystem() throws Exception {
+        final Configuration config = new Configuration();
+        EXPECTED_CONFIGURATION.forEach(config::setString);
+        final StreamExecutionEnvironment environment = new LocalStreamEnvironment(config);
+        environment.getCheckpointConfig().setCheckpointStorage("testConfig:/" + tmp.newFolder());
+        environment.enableCheckpointing(100);
+        environment.fromElements(1, 2, 3, 4).map((MapFunction<Integer, Integer>) value -> value);
```

Comment:
Is this cast needed?

Reply:
It is not about the cast but rather the execution, without inserting the mapper the job is not executable.